### ISSUE TITLE
Yaml Import for 5.0 Doesn't Handle % in URL

### ISSUE THREAD

**Issue by Ionshard** [franc6/ics_calendar#148]
Downloaded the 5.0 release to test the other issue I have. 

Started getting a "Failed to open url" error

My calendar is a private Google link of my work calendar like:

https://calendar.google.com/calendar/ical/victor.ling%40company.com/private-jdnebjsjsjdnrhsusuhe/basic.ics

(Not a working link has been obfuscated)

Which seems to be imported as https://calendar.google.com/calendar/ical/victor.ling%2540company.com/private-jdnebjsjsjdnrhsusuhe/basic.ics

So it looks like the % in my already encoded URL got re-encoded on import.

**Comment by pbaylas** [comment]
I'm getting the same problem, although it seems to have mangled the % in the url even more. My ICS url (from Proton Calendar) contains `%3D%3D` but the "Failed to open url" error is showing that as `%2525252525252525252525252525252525253D%2525252525252525252525252525252525253D`.

**Comment by franc6** [comment]
This happens due to PR #133, which shouldn't be used anymore, and I'd forgotten about it.  The worse problem reported by @philipdouglas happens because I mangled PR #133.

I should have a fix soon.

**Comment by franc6** [comment]
CORRECTION: I'm seeing another problem that occurs without fixing this one, and it's a big one.  Please do not update to 5.0.0 if you have not already.  If you have, please restore your YAML config and move back to 4.2.0 for now.  I'll have an update ASAP, but that probably means at least next week.

**Comment by plutomedia987** [comment]
This issue still exists for me. The % in the url is being replaced with %25. 

Edit:
I have got around it by replacing the pre encoded url from google. %40 was being replaced with %2540. I have manually adjusted the URL to replace the %40 with an @ symbol and your url encoder has replaced correctly.

For reference anyone else coming across this. URL encoding parameters are here:
https://www.w3schools.com/tags/ref_urlencode.ASP
